modules/social_features/social_search/modules/social_search_autocomplete/css/search-suggestion.css

```diff
@@ -0,0 +1,75 @@
+.search-suggestions {
```

Comment:
Yes! The base styling included here is what I believe is the minimum required for this module to function. Therefore I think it belongs in the module and not in a theme. If we'd want to use this in a different theme for example then it would break without this CSS.

Any CSS that's written in a theme should be loaded after this and should just overwrite this so I don't see any difficulties there.

I agree with the statement that we don't do this (yet/a lot) but I believe that to be a problem, not an example.